src/stories/containers/auth/change-password/change-password.mvvm.ts

```diff
@@ -39,6 +41,7 @@ export const userChangePasswordMvvm = () => {
   const [error, setError] = useState<string>('');
   const [isWrongOldPassword, setIsWrongOldPassword] = useState<boolean>(false);
   const [loading, setLoading] = useState<boolean>(false);
+  const isMobileOrTable = useMediaQuery(lightTheme.breakpoints.between('table_375', 'table_834'));
```

Comment:
if this is for all devices with small resolutions, it should be `lightTheme.breakpoints.down('table_834')` so this can cover resolutions from 0-365